src/groups_v2/model.rs

```diff
@@ -54,6 +54,34 @@ impl PartialEq for RequestingMember {
     }
 }
 
+#[derive(Derivative, Clone)]
+#[derivative(Debug)]
+pub struct BannedMember {
+    pub uuid: Uuid,
```

Comment:
Should this be Aci?

Reply:
I think so yes. Parts of the code path accept ServiceId when creating BannedMember (MemberBanned) in SA, but in all concrete cases I could only find use of UUIDs or ACIs.